Helper/RuleConverter.php

```diff
@@ -0,0 +1,121 @@
+<?php
+/**
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade this module to newer
+ * versions in the future.
+ *
+ *
+ * @category  Smile
+ * @package   Smile\ElasticsuiteTargetRule
+ * @author    Richard BAYET <richard.bayet@smile.fr>
+ * @copyright 2017 Smile
+ * @license   Open Software License ("OSL") v. 3.0
+ */
+
+namespace Smile\ElasticsuiteTargetRule\Helper;
+
+use Magento\Framework\App\Helper\AbstractHelper;
+use Magento\Framework\App\Helper\Context;
+use Smile\ElasticsuiteCatalogRule\Model\RuleFactory;
```

Comment:
But my point is that, *at the moment*, it is not possible to disable VirtualCategories anyway, because :
- \Smile\ElasticsuiteTargetRule\Model\Actions\Condition\Product\Attributes 
- \Smile\ElasticsuiteTargetRule\Model\Actions\Condition\Product\Special\Price
both extends \Smile\ElasticsuiteVirtualCategory\Model\Rule\Condition\Product

Cheers